c2rust-analyze/src/analyze.rs

```diff
@@ -2269,7 +2281,7 @@ fn pdg_update_permissions<'tcx>(
                     perms.insert(PermissionSet::OFFSET_SUB);
                 }
                 if !node_info.unique {
-                    perms.remove(PermissionSet::UNIQUE);
+                    //perms.remove(PermissionSet::UNIQUE);
```

Comment:
Should this be enabled for `!skip_borrowck_everywhere`?

Reply:
Fixed in 0ec40f6.

I originally developed this branch by just commenting out a bunch of borrowck/`UNIQUE`-related code.  I guess I missed this part when converting it to be toggleable based on a command-line option.